content/2022/videookulografija.md

```diff
@@ -1,4 +1,561 @@
+
+
+![Koordinate x-ose u odnosu na t - slika 15](/images/2022/videookulografija/filtriranx.jpg)(slika 13)
+
+
+![Koordinate y-ose u odnosu na t - slika 16](/images/2022/videookulografija/filtrirany.jpg)(slika 14)
+
+
+
+### **3.2.3. Kalibracija**
+
+
+Kada je zenicu detektovana potrebno je uraditi test gde se na ekranu pojavljuje devet tačaka u koje se gleda određen broj sekundi.
+U toku eksperimenta dobijaju se koordinate zenice i tačke u koju trenutno gledamo, tako pravimo bazu podataka koje će nam biti potrebna za dalji rad.
+
+
+Ekran koji se gleda tokom eksperimenta izgleda kao na slici 17. 
+U toku eksperimenta crvena tačka se pomera nakon nekog vremena, tj. plave tačke postaju crvene kada je potrebno gledati u njih.
+
+
+![Ekran tokom eksperimenta - slika 17](/images/2022/videookulografija/eksperiment.jpg)(silak 17)
+
+#### 3.2.3.1. Linearna regresija
+
+
+Kako bi se uz pomoć detekcije zenice kontrolisao neki objekat na računaru potrebno je znati na osnovu koordinata centra zenice u koju tačku na ekranu je oko upereno. 
+Za to je korišćena linearna regresija, tj. polinomialna regresija jer sistem nije linearan. 
+Ovaj sistem nije linearan iz dva razloga.
+Prvi razlog je konveksni oblik oka koji izaziva nelinarno kretanje zenice, a drugi razlog je što ravan pogleda(koja je normalna na položaj lica) i ravan ekrana nisu pod pravim uglom.
+
+
+
+![Primer linearne regresije - slika 18](/images/2022/videookulografija/linearRegression.jpg)(slika 18)
+
+
+Linearna regresija predstavlja modelovanje relacija između jedne ili više zavisnih promenljivih i jedne ili više nezavisnih promenljivih. 
+Na slici 13 se može videti primer modelovanja relacije između promenljivih.
+Model linearne regresije ima oblik:
+
+
+$ \textbf{Y} = \textbf{X} \cdot b $
+
+
+Gde je Y vektor izmerenih vrednosti(u ovom slučaju koordinate pogleda na ekranu) odnosno zavisna promenljiva, X vektor inputne promenljive(u ovom slučaju koordinate zenice oka) odnosno nezavisna promenljiva  i b vektor koificijenta modela. 
+
+
+Optimalni vektor parametara $b$ dobija se iz formule za pseudo inverziju koja glasi: 
+
+
+$b = (X^T \cdot X)^{-1} \cdot Y \cdot X^{T}$ 
+
+
+Gde je X matrica koordinata zence oka, a Y matrica koordinata tačaka na ekranu u koje je predviđeno da se gleda tokom prethodno pomentuog eksperimenta.
+
+
+Ove matrice se dobijaju iz baze podataka generisane tokom prethodno pomenutog eksperimenta.
+Dobijen vektor koejficijenta modela b se koristi za izračunavanje koordinta tačaka na ekranu u koje je zenica uperena.
+Ovako dobijen vektor koeficijenta b nije deltvoran.
+Konveksni oblik oka izaziva nelinarno kretanje zenice i ugao između ravni pogleda i ravni ekrana nije normalan zbog čega je potrebno koristiti polinomialnu regresiju.
+
+
+#### 3.2.3.2. Polinomialna regresija
+
+
+Sistem nije linearan zbog konveksnog oblika oka i ugla između ravni pogleda i ravni ekrana koji nije normalan zbog čega se koristi polinomialna regresija.
```

Comment:
Ajde da ne ponavljamo ovo 3 puta. Izbrišite na jednom mestu gde mislite da je to okej